pysat/tests/test_custom.py

```diff
@@ -114,10 +127,11 @@ def test_basic_str_w_function(self):
         assert self.out.find('mult_data') > 0
         assert self.out.find('Args') > 0
         assert self.out.find('Kwargs') > 0
+        return
 
     def test_single_modifying_custom_function_error(self):
-        """Test for error when custom function loaded as modify returns a value
-        """
+        """Test for error if 'modify' custom function returns a value."""
```

Comment:
'modify' doesn't exist anymore. There are only custom functions now.
```suggestion
        """Test for error if custom function returns a value."""
```